ddtrace/internal/writer.py

```diff
@@ -250,7 +252,6 @@ def __init__(
         max_payload_size=None,  # type: Optional[int]
         timeout=agent.get_trace_agent_timeout(),  # type: float
         dogstatsd=None,  # type: Optional[DogStatsd]
-        report_metrics=False,  # type: bool
```

Comment:
We should make this change in another PR. I expect a few tracer tests to fail with this change.